com.valvesoftware.Steam.CompatibilityTool.Proton.yml

```diff
@@ -13,43 +13,62 @@ sdk-extensions:
+  ldflags: -L/app/share/steam/compatibilitytools.d/ProtonGECustom/lib
   env:
-    C_INCLUDE_PATH: /app/share/steam/compatibilitytools.d/Proton/include:/app/include
-    CPLUS_INCLUDE_PATH: /app/share/steam/compatibilitytools.d/Proton/include:/app/include
-    PKG_CONFIG_PATH: "/app/share/steam/compatibilitytools.d/Proton/lib/pkgconfig\
+    C_INCLUDE_PATH: /app/share/steam/compatibilitytools.d/ProtonGECustom/include:/app/include
+    CPLUS_INCLUDE_PATH: /app/share/steam/compatibilitytools.d/ProtonGECustom/include:/app/include
+    PKG_CONFIG_PATH: "/app/share/steam/compatibilitytools.d/ProtonGECustom/lib/pkgconfig\
                      :/app/lib/pkgconfig\
                      :/app/share/pkgconfig\
                      :/usr/lib/x86_64-linux-gnu/pkgconfig\
                      :/usr/share/pkgconfig"
-    LD_RUN_PATH: /app/share/steam/compatibilitytools.d/Proton/lib
+    LD_RUN_PATH: /app/share/steam/compatibilitytools.d/ProtonGECustom/lib
   cflags: &optflags -O2 -pipe -march=nocona -mtune=core-avx2 -mfpmath=sse -fwrapv -fno-strict-aliasing
   cflags-override: true
   cxxflags: *optflags
   cxxflags-override: true
 
 x-compat-i386-opts: &compat_i386_opts
-  ldflags: -L/app/share/steam/compatibilitytools.d/Proton/lib32 -L/app/lib32
-  prepend-path: /app/share/steam/compatibilitytools.d/Proton/bin32
+  ldflags: -L/app/share/steam/compatibilitytools.d/ProtonGECustom/lib32 -L/app/lib32
+  prepend-path: /app/share/steam/compatibilitytools.d/ProtonGECustom/bin32
   append-path: /usr/lib/sdk/toolchain-i386/bin
   env:
     CC: ccache i686-unknown-linux-gnu-gcc
     CXX: ccache i686-unknown-linux-gnu-g++
-    PKG_CONFIG_PATH: "/app/share/steam/compatibilitytools.d/Proton/lib32/pkgconfig\
+    PKG_CONFIG_PATH: "/app/share/steam/compatibilitytools.d/ProtonGECustom/lib32/pkgconfig\
                      :/app/lib32/pkgconfig\
                      :/app/share/pkgconfig\
                      :/usr/lib/i386-linux-gnu/pkgconfig\
                      :/usr/share/pkgconfig"
-    LD_RUN_PATH: /app/share/steam/compatibilitytools.d/Proton/lib32
-  libdir: /app/share/steam/compatibilitytools.d/Proton/lib32
+    LD_RUN_PATH: /app/share/steam/compatibilitytools.d/ProtonGECustom/lib32
+  libdir: /app/share/steam/compatibilitytools.d/ProtonGECustom/lib32
 
 x-proton-source: &proton_source
   type: git
-  url: "https://github.com/ValveSoftware/Proton.git"
-  tag: proton-5.13-2
-  commit: 4e84cd392d8b6c4bae0b21ef4ad5b6c86f13cb3b
+  url: "https://github.com/GloriousEggroll/proton-ge-custom.git"
+  tag: 5.21-GE-1
+  commit: f65dbb82c4c1b7f68adb23fba803a88438c17310
+
+x-vk-xml: &vk_xml
+  type: file
+  url: https://raw.githubusercontent.com/KhronosGroup/Vulkan-Docs/v1.2.160/xml/vk.xml
+  sha256: 73b449328899fb67703e9a229b6c8858f31b2933485b08710b4f69f842add94b
+
+x-proton-patches: &proton_patches
+  type: shell
+  commands:
+    - |
+      sed \
+        -e "s#\./dlls/winevulkan/make_vulkan#./dlls/winevulkan/make_vulkan --verbose --xml $(pwd)/vk.xml#g" \
```

Comment:
Maybe just put `vk.xml` to `wine/dlls/winevulkan/vk-1.2.160.xml` (or wherever `make_vulkan` expects it to be) instead editing the script?

Reply:
I would like to have the verbose flag as well.